modules/bash

```diff
@@ -16,7 +19,7 @@ bash_configure := CFLAGS="-g0 -Os" LDFLAGS="-s" ./configure \
         --mandir=/usr/share/man \
         --without-bash-malloc \
 	--disable-coprocesses \
-	--disable-debugger \
```

Comment:
@JonathonHall-Purism Please also reenable autocompletion.

Reply:
Enabling readline added 106 KB to compressed initrd (checked librem_14).

Reply:
Thats big but passes CI builds! 
@JonathonHall-Purism Do you think the autocompletion is worth 106KB? (I think I go that side to ease testing).

Reply:
Yeah, I do.  Invoking a function to test is needlessly painful from the recovery shell right now (`$ bash -c '. /etc/functions; func'`, something I seem to need to do a lot).

This is one step closer to possibly making bash the recovery shell by default and doing away with ash.  With readline back, legacy-flash boards are the only remaining holdout I think.